src/edu/wright/cs/raiderplanner/controller/ChatController.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2018 - Michael Pantoja
+ * Copyright (C) 2018 - Michael Pantoja, Cole Morgan
```

Comment:
One does not customarily add oneself as a copyright claimant for such a small change.  Generally a 10+ line change would be the minimum.